proc/proc.go

```diff
@@ -740,37 +732,21 @@ func initializeDebugProcess(dbp *Process, path string, attach bool) (*Process, e
 
 func (dbp *Process) ClearTempBreakpoints() error {
 	for _, bp := range dbp.Breakpoints {
-		if !bp.Temp {
+		if bp.Kind == NormalBreakpoint {
```

Comment:
Maybe not `Temp()` but maybe `Internal()` or something to say that it was not set by a user and is for the debuggers own internal user.